doc_source/create-managed-node-group.md

```diff
@@ -135,7 +136,7 @@ We recommend using a role that's not currently in use by any self\-managed node
    + **node group update configuration** – \(Optional\) You can select the number or percentage of nodes to be updated in parallel\. Select either **Number** or **Percentage** to enter a value\. These nodes won't be available during the update\. 
 
 1. On the **Set compute and scaling configuration** page, fill out the parameters accordingly, and then choose **Next**\.
-   + **AMI type** – Choose **Amazon Linux 2 \(AL2\_x86\_64\)** for Linux non\-GPU instances, **Amazon Linux 2 GPU Enabled \(AL2\_x86\_64\_GPU\)** for Linux GPU instances, **Amazon Linux 2 Arm \(AL2\_ARM\_64\)** for Linux Arm instances, **Bottlerocket \(BOTTLEROCKET\_x86\_64\)** for Bottlerocket x86\_64 instances, or **Bottlerocket Arm \(BOTTLEROCKET\_ARM\_64\)** for Bottlerocket Arm instances\.
+   + **AMI type** – Choose **Amazon Linux 2 \(AL2\_x86\_64\)** for Linux non\-GPU instances, **Amazon Linux 2 GPU Enabled \(AL2\_x86\_64\_GPU\)** for Linux GPU instances, **Amazon Linux 2 Arm \(AL2\_ARM\_64\)** for Linux Arm instances, **Bottlerocket \(BOTTLEROCKET\_x86\_64\)** for Bottlerocket x86\_64 instances, **Bottlerocket Arm \(BOTTLEROCKET\_ARM\_64\)** for Bottlerocket Arm instances, or **Windows 2019 Core \(WINDOWS\_CORE\_2019\_x86\_64\)** for Windows instances, replacing CORE with FULL and 2019 with 2022 based on your desired AMI type\.
```

Comment:
This line is odd. Can we just write them out for Windows? `Windows 2019 Core`, `Windows 2019 Full`, `Windows 2022 Core`, and `Windows 2022 Full` AMI types. Its less text and more clear I think